tests/test_yaml/version_pipcheck_outputs_correct.yaml

```diff
@@ -0,0 +1,133 @@
+{% set TEST_PYTHON = '${PYTHON}' %}  # [not win]
+{% set TEST_PYTHON = '%PYTHON%' %}  # [win]
+{% set name = "camb" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cmbant/CAMB/archive/{{ version }}.tar.gz
+  sha256: 3ab4eec98a3f5e97c8acef66a79778db380b66afd577712e07a342fdd6073113
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('fortran') }}  # [not win]
+    - m2w64-toolchain  # [win]
+    - git  # [not win]
+    - m2-git  # [win]
+    - make  # [not win]
+    - m2-make  # [win]
+    - python
+    - setuptools
+    - pip
+
+test:
+  imports:
+    - camb
+  files:
+    - params.ini
+    - test_pycamb.py
+    - run_test.sh  # [win]
+  commands:  # [not win]
+    - echo '1'  # [not win]
+    - '{{ TEST_PYTHON }} -m pip check'                                          # [not win]
```

Comment:
Can't you just do, `python -m pip check`?

Reply:
I thought it was python.exe on windows?

Reply:
Nevermind! This works on windows too! Thanks!

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=118793&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=7225e372-3a5e-5b11-c153-719d50bbeacd&l=967